docs/QSG/tour.md

```diff
@@ -219,13 +218,13 @@ bring-up DAOS servers and clients.
 	A::EVERYONE@:
 
 	# Update pool access entry for the existing test_group1 to no-permission
-	dmg pool update-acl -e A:G:test_group1@: --pool=$DAOS_POOL
```

Comment:
--pool is not available so can be removed
`dmg pool update-acl --help
Usage:
  dmg [OPTIONS] pool update-acl [update-acl-OPTIONS] [<pool name or UUID>]

Application Options:
      --allow-proxy              Allow proxy configuration via environment
  -l, --host-list=               A comma separated list of addresses <ipv4addr/hostname> to connect to
  -i, --insecure                 Have dmg attempt to connect without certificates
  -d, --debug                    Enable debug output
  -j, --json                     Enable JSON output
  -J, --json-logging             Enable JSON-formatted log output
  -o, --config-path=             Client config file path

Help Options:
  -h, --help                     Show this help message

[update-acl command options]
      -a, --acl-file=            Path for new Access Control List file
      -e, --entry=               Single Access Control Entry to add or update`